.github/workflows/openshift-deploy.yml

```diff
@@ -133,6 +150,10 @@ jobs:
           #Jasper variables
           JASPER_URL: https://cirras-underwriting-jasper-${{ inputs.ENVIRONMENT_NAME }}-${{ inputs.NAMESPACE }}.apps.silver.devops.gov.bc.ca/jasperserver/
           JASPER_USERNAME: ${{ vars.JASPER_USERNAME }}
+          JASPER_CPU_REQUEST: ${{ vars.JASPER_CPU_REQUEST }}
```

Comment:
I thought we don't use this action for Jasper?

Reply:
Liquibase is the one we don't use this for - Jasper is correct here